### ISSUE TITLE
Include tax on fees in the line item adjustment table on admin/orders page

### ISSUE THREAD

**Issue by tschumilas** [openfoodfoundation/wishlist#449]
## What is the need / problem?
On the Admin/orders page,  a hub manager should be able to see all the costs on an order,  and the details on this page should add up to the order total 

At present,  taxes on fees (when applicable) are not included in the line item adjustment table on the admin/orders page -  and users cannot understand why they cannot arrive at the order total from the information presented to them on this page.  This prompts a request from most new users telling us that their orders are calculating incorrectly.

## Which type of users does this problem affect (and how many, if known)?
_e.g._
Hub Managers,  and Producers with their own shop.  Any user who sells taxable products,  and also uses fees.

## Potential solutions that will solve the problem  ?[[brainstorming to list feature candidates]
The taxes on fees are calculating correctly.  Order confirmations at checkout and invoices,  and reports are all correct.

Can we add the taxes on fees (when applicable) to the line item adjustment table on the admin/orders page?

## Additional context
This shop has a 20% enterprise fee on all products.
This fee is set to 'taxable' (but same result if it is set to 'inherit from product'
They sell a taxable product - candles ($10)
I purchase 1 candle and the costs should be:
Product - $10
Tax on product (13% x $10) = $1.30
Fee on product (20% x %10) = $2.00
Fee on product tax (13% x $2.00) = $0.26
Order total is $13.56,   Including tax of $1.56

See below that the confirmation at checkout to the customer is correct,  and the invoice is correct...
But see that the hub manager is confused because they do not arrive at the order total if they add up the product + line items on the admin/orders page.
![Checkout-Open-Food-Network](https://github.com/openfoodfoundation/wishlist/assets/13171778/bf36627e-b70a-4fa8-b6c9-2dc2670230de)
![Orders-OFN-Administration (1)](https://github.com/openfoodfoundation/wishlist/assets/13171778/5e84ec29-3211-4dd5-983b-b98bbaed2867)

**Comment by RachL** [comment]
@mkllnk we need your view on that one: do you think it could be done as a papercut?

**Comment by mkllnk** [comment]
Yes, it probably needs an hour or two to understand the data structure and to find the best solution but then it should easy to change. We currently just list adjustment associated to line items but we need to add the adjustments of those adjustments as well. Just go a level deeper.

I think that we need to change this line:
https://github.com/openfoodfoundation/openfoodnetwork/blob/7ef4df008729c0a758a5d2f4512b3ccbc453ffd3/app/views/spree/admin/orders/_form.html.haml#L14

We can add a helper like the one for order adjustments:
https://github.com/openfoodfoundation/openfoodnetwork/blob/7ef4df008729c0a758a5d2f4512b3ccbc453ffd3/app/helpers/admin/orders_helper.rb#L5-L11

It needs to add the tax adjustments to the current list.
